lib/krb5/pac.c

```diff
@@ -2032,8 +2032,8 @@ _krb5_pac_get_attributes_info(krb5_context context,
     return 0;
 }
 
-static unsigned char single_zero = '\0';
-static krb5_data single_zero_pac = { 1, &single_zero };
+static const unsigned char single_zero = '\0';
+static const krb5_data single_zero_pac = { 1, rk_UNCONST(&single_zero) };
```

Comment:
shouldn't this be krb5_const_data?

Reply:
Sorry, there are so many other krb5_const types these days I simply assumed!  

This change seems correct then.